packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseInActionItem.tsx

```diff
@@ -150,13 +153,20 @@ export const SynapseInActionItem: React.FunctionComponent<
           display: { xs: 'none', md: 'block' },
           justifySelf: 'end',
           alignSelf: 'end',
-          opacity: opacity,
         }}
       >
-        <ImageFromSynapseTable
-          tableId={tableId}
-          fileHandleId={imageFileHandleId}
-        />
+        <Fade in={inView} timeout={transitionTimeoutMs}>
+          <div>
```

Comment:
Div wrapper to avoid open MUI issue https://github.com/mui/material-ui/issues/27154